forge-gui-desktop/src/main/config/forge.sh

```diff
@@ -38,7 +38,7 @@ jdk_version() {
 }
 v="$(jdk_version)"
 
-SHAREDPARAMS='-Xmx4096m -Dfile.encoding=UTF-8 -jar $project.build.finalName$'
```

Comment:
This isn't quite right. It doesn't seem to be escaping spaces right

Reply:
:+1: